deployments/peerscout/peerscout--stg.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: peerscout--stg
+  namespace: peerscout
+  annotations:
+    fluxcd.io/automated: 'true'
+    filter.fluxcd.io/chart-image: glob:*
+spec:
+  releaseName: peerscout--stg
+  chart:
+    git: https://github.com/elifesciences/data-science-dags
+    path: charts/peerscout
+    ref: develop
+
+  values:
+    image:
+      repository: elifesciences/data-science-dags_peerscout-api_unstable
+      tag: 39aa9515d6bc752e246f5715c468f891135675dc
+    env:
+      DATA_SCIENCE_STATE_PATH: s3://staging-elife-data-pipeline/airflow-config/data-science/state
+      GOOGLE_APPLICATION_CREDENTIALS: /root/.gcp/credentials.json
```

Comment:
Not a blocker, but I do wonder depending on the library used if there is a default for this and if so, where does it point and is this var needed.

Reply:
A quick read up showed it's required for the client library when not running on GCP with default service credentials: https://cloud.google.com/docs/authentication/production